docs/content/operations/including-extensions.md

```diff
@@ -3,31 +3,47 @@ layout: doc_page
 ---
 # Including Extensions
 
-Druid uses a module system that allows for the addition of extensions at runtime.
+Druid uses a module system that allows for the addition of extensions at runtime. To instruct Druid to load extensions, follow the steps below.
 
-## Specifying extensions
+## Download extensions
 
-Druid extensions can be specified in the `common.runtime.properties`. There are two ways of adding druid extensions currently.
+Core Druid extensions are already bundled in the Druid release tarball. You can get them by downloading the tarball at [druid.io](http://druid.io/).
+Unpack the tarball, you will see an ```extensions``` folder that contains all the core extensions, along with a ```hadoop-dependencies``` folder
+where it contains all the hadoop extensions. Each extension will have its own folder that contains extension jars. However, due to license issue,
+we didn't package `mysql-metadata-storage` extension in it, in order to get it, you can download it from [druid.io](http://druid.io/), unpack it and then
+move it into ```extensions``` directory.
 
-### Add to the classpath
+Optionally, you can leverage the `pull-deps` tool to download extensions you want. 
+See [pull-deps](../operations/pull-deps.html) for a complete example.
 
-If you add your extension jar to the classpath at runtime, Druid will load it into the system.  This mechanism is relatively easy to reason about, but it also means that you have to ensure that all dependency jars on the classpath are compatible.  That is, Druid makes no provisions while using this method to maintain class loader isolation so you must make sure that the jars on your classpath are mutually compatible.
+## Load extensions
 
-### Add to the extension directory
+There are two ways of Druid loading extensions.
 
-If you don't want to fiddle with classpath, you can create an extension directory and tell Druid to load extensions from there.
+### Load from classpath
+
+If you add your extension jar to the classpath at runtime, Druid will load it into the system.  This mechanism is relatively easy to reason about, 
+but it also means that you have to ensure that all dependency jars on the classpath are compatible.  That is, Druid makes no provisions while using 
+this method to maintain class loader isolation so you must make sure that the jars on your classpath are mutually compatible.
+
+### Load from extension directory
+
+If you don't want to fiddle with classpath, you can tell Druid to load extensions from an extension directory.
```

Comment:
It would be nice here to say that you are going to talk about two ways to dell Druid where to load extensions from.